### ISSUE TITLE
High contrast mode for better a11y

### ISSUE THREAD

**Issue by devtobi** [it-at-m/refarch-templates#87]
**Is your feature request related to a problem? Please describe.**
In order to improve a11y and readability there should a possibility to enable a "high contrast" mode. This should be a custom theme with improved readability

**Describe the solution you'd like**
Simple VueComponent that uses localStorage (e.g. via VueUse) to store selected mode. On change the current vuetify theme should be changed dynamically. This can be achieved by defining a custom Vuetify theme.
On top of that, the browser settings can be accessed via https://vueuse.org/core/usePreferredContrast/ composable.

**Describe alternatives you've considered**
---

**Additional context**
See https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-apps/packages/apps/digiwf-tasklist/src/components/UI/help/AppHelpMenu.vue#L19 and https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-apps/packages/apps/digiwf-tasklist/src/plugins/vuetify.ts for an example implementation
